command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
@mikebrow How will the compliance testing tool know how to create a container and validate it works against a runtime with no known API? How do you propose writing the test/certification tool unless we specify the API it can expect to validate against? How does OCI-tool know how to invoke a web API it has never heard of?

Yes, the original reason we hesitated to specify the cli was worries it'd limit ux choices, the reason we went back on that was realising that higher level things - like compliance tools and orchestrators - *need* an API in order to actually interact with things and allow compliant runtimes to be swapped in and out (which to me is kind of the point of a spec)